multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/metadata/util/MtaMetadataUtil.java

```diff
@@ -45,6 +47,7 @@ public static Metadata getMetadataWithoutMtaFields(Metadata metadata) {
                        .from(metadata)
                        .label(MtaMetadataLabels.MTA_ID, null)
                        .label(MtaMetadataLabels.MTA_NAMESPACE, null)
+                       .label(MtaMetadataLabels.SPACE_GUID, null)
```

Comment:
Why is the space guid added and/or needed?

Reply:
It's needed for the DetectDeployedMtaStep as serviceKeys don't have a filter by spaceGuid in the v3 api - we add that as a label and select by that later